src/utils/L.Map.Sync.js

```diff
@@ -0,0 +1,91 @@
+import L from 'leaflet'
+
+const noAnimate = { animate: false }
+
+L.Map.include({
+    _syncedMaps: {},
+    _onDragEndSync: {},
+
+    sync: function(id) {
+        const { setView } = L.Map.prototype
+        const origin = this
+        let syncedMaps = this._syncedMaps[id]
+
+        if (!syncedMaps) {
+            syncedMaps = []
+            this._syncedMaps[id] = syncedMaps
```

Comment:
Minor cleanup - this could maybe be a little cleaner without the `let`, like so:

```js
if (!this._syncedMaps[id]) {
  this._syncedMaps[id] = []
}
const syncedMaps = this._syncedMaps[id]
```

Reply:
Changed!